indico/modules/events/registration/controllers/management/regforms.py

```diff
@@ -252,6 +255,37 @@ def _process(self):
                                                     regform=self.regform)
 
 
+class RHRegistrationFormNotificationPreview(RHManageRegFormBase):
+    """Preview registration emails."""
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.commit = False
+
+    @no_autoflush
+    def _process(self):
+        message = request.form.get('text', '')
```

Comment:
Let's use webargs (`@use_kwargs`) for this:

```py
@use_kwargs({
    'message': fields.String(load_default=''),
})
@no_autoflush
def _process(self, message):
    ...
```

Reply:
Done 👍